src/import.js

```diff
@@ -74,12 +74,18 @@ export default function up() {
           type: 'string',
           default: undefined,
         })
+        .option('headers-file', {
+          alias: 'headersFile',
+          describe: 'File location for your .json file for headers to be used in the requests',
```

Comment:
```suggestion
          describe: 'Location of a custom .json file containing headers to be used with all proxy requests',
```

Reply:
addressed in https://github.com/adobe/helix-cli/pull/2368/commits/0a07dc030893c6224008a1ec5007290065756363